src/bin/bat/app.rs

```diff
@@ -322,7 +322,7 @@ impl App {
                             .collect::<Vec<_>>()
                     })
                     .or(env_style_components)
-                    .unwrap_or_else(|| vec![StyleComponent::Full])
+                    .unwrap_or_else(|| vec![StyleComponent::Auto])
```

Comment:
Not showing all header fields by default has surfaced a somewhat thorny problem.

First of all, we don't want to change this to `Auto`. Wether or not to show style components is and should remain to be `--decorations` which defaults to `auto`.

It might very well make sense to remove `StyleComponent::Auto`. At least from the CLI, to remain semver compatible with `bat` as a library. See https://github.com/sharkdp/bat/issues/1743 and linked issues.